utils/utils.go

```diff
@@ -219,6 +220,38 @@ func CopyFileIfExists(source, dest string) error {
 	}})
 }
 
+func CopyToTempFile(sourceFileName, directory, pattern string) (string, error) {
```

Comment:
maybe you can use cp.Copy like in 
https://github.com/openshift-kni/lifecycle-agent/pull/431/commits/f8808ac7bb220896af3d7b960dce4a56039b3756#diff-0df82f8689d3e8d881be0653ab4e9f8221c5835ec95a14930d11669e9c7f07efR215
it will minimize this function no?

Reply:
nvm... I guess that wouldn't exist in IBI

Reply:
Note added :+1: